backend/packages/identity-scan/src/scan/events/identity/constants.js

```diff
@@ -0,0 +1,14 @@
+// constants.js
+module.exports = {
+  IDENTITY: "identity",
+  IDENTITY_SET: "IdentitySet",
+  IDENTITY_CLEARED: "IdentityCleared",
+  IDENTITY_KILLED: "IdentityKilled",
+  JUDGEMENT_GIVEN: "JudgementGiven",
+  JUDGEMENT_REQUESTED: "JudgementRequested",
+  JUDGEMENT_UNREQUESTED: "JudgementUnrequested",
+  REGISTRAR_ADDED: "RegistrarAdded",
+  SUB_IDENTITY_ADDED: "SubIdentityAdded",
+  SUB_IDENTITY_REMOVED: "SubIdentityRemoved",
+  SUB_IDENTITY_REVOKED: "SubIdentityRevoked",
+};
```

Comment:
It's better if we can classify upper constants. `IDENTITY` is a module name, but others are events name.